cmd/tailscale/cli/set.go

```diff
@@ -128,3 +136,27 @@ func runSet(ctx context.Context, args []string) (retErr error) {
 	_, err = localClient.EditPrefs(ctx, maskedPrefs)
 	return err
 }
+
+func calcAdvertiseRoutesForSet(advertiseExitNodeSet, advertiseRoutesSet bool, curPrefs *ipn.Prefs, setArgs setArgsT) (routes []netip.Prefix, err error) {
```

Comment:
this signature could use some docs, even if it's unexported

Reply:
Done